docs/glossary.md

```diff
@@ -151,7 +151,7 @@ _Tags: [term](/tags/term), [explanation](/tags/explanation), [delay-workflow](/t
 
 #### [Dual Visibility](/visibility#dual-visibility)
 
-Dual Visibility is a feature that lets you set a secondary Visibility store in your Temporal Cluster to facilitate migrating your Visibility data from one database to another.
+Dual Visibility is a feature that lets you set a secondary Visibility store in your Temporal Service to facilitate migrating your Visibility data from one database to another.
```

Comment:
```suggestion
Dual Visibility is a feature, specific to a Self-hosted Temporal Service, that lets you set a secondary Visibility store in your Temporal Service to facilitate migrating your Visibility data from one database to another.
```

Reply:
Or, "that lets you set a secondary Visibility store in your Self-hosted Temporal Service"

Just something to indicate this is specific to Self-hosting.